panther_ioc_rules/atlassian_confluence_ip_iocs.py

```diff
@@ -0,0 +1,10 @@
+from panther_iocs import VOLEXITY_CONFLUENCE_IP_IOCS, ioc_match
+
+
+def rule(event):
+    return any(ioc_match(event.get("p_any_ip_addresses"), VOLEXITY_CONFLUENCE_IP_IOCS))
+
+
+def title(event):
+    ips = ",".join(ioc_match(event.get("p_any_ip_addresses"), VOLEXITY_CONFLUENCE_IP_IOCS))
+    return f"IP seen in May 2022 exploiting Confluence 0-Day: {ips}"
```

Comment:
The phrasing here seems a little awkward, maybe

```suggestion
    return f"IP seen from May 2022 exploitation of Confluence 0-Day: {ips}"
```

With phrasing it kind be very subjective so feel free to disregard this comment.